CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+## How to contribute to simplectemplate
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+Please note we have a Pull Request Process, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+* Creating a new branch off of the develop branch
+* All changes to the code should have appropriate unit tests
+* All code should match PSR-2 style
+* Submit a pull request again develop as the base branch
+* All travis checks will need to pass
```

Comment:
Again, above is not applicable right now. I have created #12 and will add after that is complete.